c++/src/kj/async.c++

```diff
@@ -1311,7 +1313,7 @@ const Executor& getCurrentThreadExecutor() {
 
 namespace _ {  // private
 
-#if !(_WIN32 || __CYGWIN__ || __BIONIC__)
+#if KJ_USE_FIBERS
```

Comment:
Looks like the meaning of the check changed here? `_WIN32` and `__CYGWIN__` should avoid this block.

Reply:
Fixed.